### PR TITLE
Terminal title

### PR DESCRIPTION
**Pull request by Arique1104**

### CODE REVIEW

**Comment by Arique1104** [comment]
The main menu, and winner banners are now very obvious and very magenta.

**Comment by priyapower** [comment]
<3 these banners and obvious magenta!

File: lib/game.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 require_relative "./battleship"
+require "colorize"
```
Comment (priyapower) [inline lib/game.rb:2]: <3

File: lib/game.rb [hunk 2]
```diff
@@ -73,8 +75,25 @@ def start
     end_game
   end
 
+  def print_welcome_to_battleship
```
Comment (priyapower) [inline lib/game.rb:78]: This is incredible work. I can see how hard you worked on this and I love that our game will feel like us!

File: test/game_test.rb [hunk 3]
```diff
@@ -236,6 +237,13 @@ def test_it_can_get_auto_coordinates
     # assert_equal "M", @game.auto.board.cells["B2"].render
   end
 
+  def test_it_can_print_banners
```
Comment (priyapower) [inline test/game_test.rb:240]: Thank you for testing your helper methods.

File: test/game_test.rb [hunk 4]
```diff
@@ -250,21 +258,22 @@ def test_it_can_setup_ships
     user_board = Board.new
     user_board.generate_cells
 
-    auto = Player.new(:auto, auto_board)
-    user = Player.new(:user, user_board)
+    auto = mock("Auto Player", auto_board)
```
Comment (priyapower) [inline test/game_test.rb:261]: Mocks!!!!! :)

**Pull request merged**